lattice-controller/Makefile

```diff
@@ -0,0 +1,14 @@
+# capability-provider/lattice-controller/Makefile
+
+PROJECT = lattice-controller
+CAPABILITY_ID = wasmcloud:latticecontrol
+VENDOR   = "wasmCloud"
+VERSION  = $(shell cargo metadata --no-deps --format-version 1 | jq -r '.packages[] .version' | head -1)
+REVISION = 0
+oci_url  = localhost:5000/v2/$(PROJECT):$(VERSION)
+
+include ../build/makefiles/provider.mk
+
+test:
+	killall target/debug/kvredis || true
```

Comment:
this looks autogenerated but I think it should be `target/debug/latticecontroller` or whatever the resulting binary is